.gitignore

```diff
@@ -0,0 +1 @@
+<img src="https://prd-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/styles/carousel_large/public/thumbnails/image/wss-wu-dryville.jpg?itok=uxY0Fwj1"/>
```

Comment:
This should just be the filepath. In this case, `wss-wu-dryville.jpg`

Reply:
Oh I see, I misunderstood the directions, that's my fault. I should be able to quickly redo it, would it be best to open a new PR in that case?

Reply:
You can continue adding to this PR! Adding another commit to the branch will also add the commit to this PR